mailbox/hbase/src/main/java/org/apache/james/mailbox/hbase/mail/HBaseMailboxMessage.java

```diff
@@ -354,6 +356,10 @@ public int compareTo(MailboxMessage other) {
 
     @Override
     public List<MessageAttachment> getAttachments() {
-        throw new NotImplementedException("Attachments are not implemented");
+        try {
```

Comment:
why not a default method?

Reply:
because I would not be in MailboxMessage anyway because it depends on mailbox-store.
I hesitate to create a "mixin interface" for that but I think it's too complicated.